source/core/VirtualInstrument.cpp

```diff
@@ -621,6 +621,12 @@ void ClumpParseState::AddDataTargetArgument(SubString* argument, Boolean addType
         // StaticTypeAndData formal parameters get passed the type and pointer to the data.
         // they are fully polymorphic.
         InternalAddArg(null, ActualArgumentType());
+        static SubString etad("EnumTypeAndData");
```

Comment:
etad? why not enumTypeAndData? As the code evolves, variable definitions tend to get separated from actual use, making harder to undertand the meaning of a variable.